msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -418,6 +418,49 @@ public async Task<AuthenticationResult> AcquireTokenAsync(IEnumerable<string> sc
                         behavior, extraQueryParameters, parent, ApiEvent.ApiIds.AcquireTokenWithScopeUserBehaviorAuthority).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Non-interactive request to acquire token via Kerberos / Windows Integrated Authentication.
```

Comment:
Non-interactive request to acquire a security token for the signed-in user in Windows, via Integrated Windows Authentication. See https://aka.ms/msal-net-iwa.
The account used in this overrides is pulled from the operating system as the current user principal name